projects/core/components/primitive-textfield/primitive-textfield.component.ts

```diff
@@ -36,8 +36,11 @@ import {map} from 'rxjs/operators';
 
 import {TuiPrimitiveTextfield} from './primitive-textfield-types';
 
-const ICON_PADDING_L = 1.75;
-const ICON_PADDING = 1.625;
+export const TUI_ICON_PADDINGS: Record<TuiSizeS | TuiSizeL, number> = {
```

Comment:
Do we need use DI for this case in the future?

Reply:
No, we do not want this to be configurable.